src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,101 @@
+  const [undoClicked, setUndoClicked] = useState(false);
+  useEffect(() => {
+    setArticleVotes(votes);
+  }, [votes]);
+
+  const handleClickUpvote = () => {
+    setArticleVotes(articleVotes + 1);
+    setErr(null);
+    setUpvoteClicked(true);
+    setUndoClicked(false);
+    upvoteArticle(articleId, { inc_votes: 1 }).catch((err) => {
+      setArticleVotes(articleVotes);
+      setUpvoteClicked(false);
+      setErr("Cannot change votes. Please try again");
+    });
+  };
+  const handleClickDownvote = () => {
+    setArticleVotes(articleVotes - 1);
+    setErr(null);
+    setDownvoteClicked(true);
+    setUndoClicked(false);
+    upvoteArticle(articleId, { inc_votes: -1 }).catch((err) => {
+      setArticleVotes(articleVotes);
+      setDownvoteClicked(false);
+      setErr("Cannot change votes. Please try again");
+    });
+  };
+
+  const handleClickUndoUpvote = () => {
+    setArticleVotes(articleVotes - 1);
+    setErr(null);
+    setUndoClicked(true);
+    setUpvoteClicked(false);
+    upvoteArticle(articleId, { inc_votes: -1 }).catch((err) => {
+      setArticleVotes(articleVotes);
+      setDownvoteClicked(false);
+      setErr("Cannot change votes. Please try again");
+    });
+  };
+  const handleClickUndoDownvote = () => {
+    setArticleVotes(articleVotes + 1);
+    setErr(null);
+    setUndoClicked(true);
+    setDownvoteClicked(false);
+    upvoteArticle(articleId, { inc_votes: -1 }).catch((err) => {
+      setArticleVotes(articleVotes);
+      setDownvoteClicked(false);
+      setErr("Cannot change votes. Please try again");
+    });
+  };
+
+  return (
+    <>
+      {" "}
+      <Box className="individual-article-box">
+        <Typography variant="h6">
+          <>Votes: {articleVotes}</>
+        </Typography>
+      </Box>
+      <Box className="individual-article-box">
+        {" "}
+        {upvoteClicked === false && downvoteClicked === false ? (
+          <>
+            <button onClick={handleClickUpvote}> UPVOTE ARTICLE (+1)</button>
+          </>
+        ) : downvoteClicked === true && upvoteClicked === false ? (
+          <button disabled={true} onClick={handleClickUpvote}>
+            {" "}
+            UPVOTE ARTICLE (+1)
+          </button>
+        ) : (
+          <button onClick={handleClickUndoUpvote}> UNDO</button>
+        )}
+        {err ? <p>{err}</p> : null}
+      </Box>
+      <Box className="individual-article-box">
+        {" "}
+        {downvoteClicked === false && upvoteClicked === false ? (
+          <button onClick={handleClickDownvote}> DOWNVOTE ARTICLE (+1)</button>
+        ) : downvoteClicked === false && upvoteClicked === true ? (
+          <button disabled={true} onClick={handleClickDownvote}>
+            {" "}
+            DOWNVOTE ARTICLE (+1)
+          </button>
+        ) : (
+          <button onClick={handleClickUndoDownvote}> UNDO</button>
+        )}
+        {err ? <p>{err}</p> : null}
+      </Box>
+    </>
```

Comment:
When the page loads, we have these rogue `null`s here, can you remove these?
<img width="233" alt="image" src="https://github.com/tjhind/tiahs-news/assets/9512932/4890cea8-d936-4081-b60c-9fbe2c1dcf85">